src/super_gradients/recipes/imagenet_regnetY.yaml

```diff
@@ -1,14 +1,26 @@
-#  RegnetY800 Imagenet classification training:
-#  This example trains with batch_size = 128
-#  Training time on a single GeForce RTX 2080 Ti is ~ 44min / epoch, total time ~ 2d 22h 50m.
-#  Reach => 76.05 (Final averaged model)
-#  Log and tensorboard at s3://deci-pretrained-models/regnetY800-imagenet-tensorboard/
+#  RegnetY Imagenet classification training:
+#  This example trains with batch_size = 256
+#  Training time on a single GeForce RTX 2080 Ti, and top1 accuracies:
+#  11 days for RegnetY200, 70.88
+#  12 days for RegnetY400, 74.74
+#  19 days for RegnetY600, 76.18
+#  20 days for RegnetY800, 77.07
+#  NOTE: Training should probably be lower as resources were shared among the above runs.
```

Comment:
I think most users would like to parallel train this. You might want to add a comment stating is this is possible or not and how.

Reply:
We haven't tested that if we parallelise we reproduce these results...so I think it's a bad idea to add such comment. If a user wants to try and reproduce theses results while using DDP, I think it should be at their own responsibility.